single-page-application/cypress/component/ShowError.cy.jsx

```diff
@@ -26,5 +26,14 @@ describe('compliance', () => {
       cy.get('button').click();
       cy.get('div[role="alert"]').should('not.exist');
     });
+    it('translates exceptions to error messages', () => {
+      cy.mount(
+        <ErrorDisplayBoundary>
+          <PutErrorInContext error={new Error('{"type":"StatusRuntimeException","message":"NOT_FOUND: no such list"}')} />
+          <ShowsErrorFromContext />
+        </ErrorDisplayBoundary>,
+      );
+      cy.get('div').should('contain.text', 'no list of such name exists');
```

Comment:
and i'd rather map the JSON exception to a different error message in the frontend